internal/configs/resource.go

```diff
@@ -68,6 +68,7 @@ type ManagedResource struct {
 	Connection   *Connection
 	Provisioners []*Provisioner
 
+	lcContent           *hcl.BodyContent
```

Comment:
I would prefer something more explicit like `lifecycleBody`.

`Content` is more abstract IMO and `lc` doesn't add much context outside of this issue. I would need to check usages of this variable to ensure I understand correctly.